plugins/out_file/file.c

```diff
@@ -152,6 +154,13 @@ static int cb_file_init(struct flb_output_instance *ins,
         ctx->label_delimiter = ret_str;
     }
 
+    tmp = flb_output_get_property("line_append", ins);
+    if (tmp) {
+        if (!strcasecmp(tmp, "true")) {
+            ctx->line_append    = 1;
+        }
+    }
```

Comment:
I think these lines can be removed since config_map backend will do.
https://github.com/fluent/fluent-bit/blob/v1.9.0/src/flb_config_map.c#L149-L157

Reply:
Thank you for this good advise.